app/gateway/database/postgres/accounts/accounts.go

```diff
@@ -1,17 +1,16 @@
 package postgres_account
 
 import (
+	"database/sql"
 	"stoneBanking/app/domain/entities/account"
-
-	"github.com/jackc/pgx/v4"
 )
 
 type accountRepository struct {
-	db *pgx.Conn
+	db *sql.DB
```

Comment:
Porque tu achou melhor usar o pacote `sql` ao `pgx`?

Reply:
Por sql ser pacote nativo, fica menos dependente de implementações externas. Igualmente, torna mais fácil caso surja a necessidade de troca de banco de dados (neste caso basta trocar o plugin e fazer os ajustes onde for necessário nas querries)

Reply:
Interessante escolha e bons pontos. Acredito que talvez fosse legal tu trocar pro `pgx` pra tu aprender a usar essa biblioteca que usamos, acho que seria bom perguntar ao Bruno

Reply:
Em termos sim, mas a alteração não traria tanto mais conhecimento. Tanto que a prova disso foi que troquei facilmente pelo sql (retirando a variavel de contexto das consultas e mudando de conn para db, nas passagens). Logo, ao menos no momento, não consigo ver onde mais agregaria usar no momento a pgx (lembrando que já havia feito ela, e  a versão original com pgx).

Reply:
Tranquilo mah, por enquanto os métodos são os mesmos, mas se tu olhar a doc dos dois tu vê que é diferente. Mas acredito que tu não vai precisar nada especifico mesmo

Reply:
Obrigado, não vou deixar de estudar ela, pode ficar tranquilo.